src/api/admin_user/handlers/admin_user_table_handler.rs

```diff
@@ -47,4 +47,5 @@ pub async fn admin_user_table_handler(
 pub struct AdminUserTableViewModel {
     pub logged_in_user: AdminUserModel,
     pub admin_user_pagination: Pagination,
+    pub success_message: String
```

Comment:
On line no #17 you have to add `mut` keyword

    pub async fn admin_user_table_handler(
        mut session: AvoRedSession,
        Query(query_param): Query<AdminUserTableQuery>,
        state: State<Arc<AvoRedState>>,
      ) -> Result<impl IntoResponse> {